agents/ocf/HealthIOWait.in

```diff
@@ -209,5 +210,3 @@ esac
 rc=$?
 ocf_log debug "${OCF_RESOURCE_INSTANCE} $__OCF_ACTION : $rc"
 exit $rc
-
-# vim: set filetype=sh expandtab tabstop=4 softtabstop=4 shiftwidth=4 textwidth=80:
```

Comment:
ditto, modeline; supposed to prevent exactly the kind of formatting issues that you're fixing here, if people are using vim to edit the file